web_src/less/_markup.less

```diff
@@ -0,0 +1,496 @@
+.markup:not(code) {
```

Comment:
Why does it need a`:not()`? I'd prefer not to have that here. It adds CSS specificity (which makes styles harder to override) not to mention probably a few hundret bytes of CSS.

Reply:
I have to agree with @silverwind here. Why is there a `:not(code)` here? There must be some way to prevent this from being needed.

Reply:
Actually I think this is existing baggage with the `.markdown:not(code)` selector. Not ideal that it's there, but no dealbreaker I guess.